manifest.konnector

```diff
@@ -22,11 +22,18 @@
       "openid+toutatice+offline_access"
     ]
   },
+  "folders": [{"defaultDir": "$Settings/$Home"}],
```

Comment:
are we sure that these variables are handled by Harvest? 🤔

Reply:
Are you sure that $Settings is handled by Harvest? https://github.com/cozy/cozy-libs/blob/master/packages/cozy-harvest-lib/src/helpers/konnectors.js#L232